packages/e2e/tests/cards/binLookup/binLookup.test.js

```diff
@@ -30,7 +30,7 @@ fixture`Testing binLookup response`
     .clientScripts('binLookup.clientScripts.js')
     .requestHooks(logger);
 
-test('Enter number of known dual branded card, ' + 'then inspect body for expected properties ', async t => {
+test.skip('Enter number of known dual branded card, ' + 'then inspect body for expected properties ', async t => {
```

Comment:
Is this accidental?

Reply:
Nope. binLookup.test.js is for v1. We now have binLookup.v2.test.js. 
Once all is confirmed and we know we're not switching back to v1 then the test file can go